src/steps/owners/converters.ts

```diff
@@ -24,9 +28,29 @@ export function createOwnerEntity(data: Owner) {
         // We may or may not need "userId" - we'll have to see later depending on the other resources and their connections
         // Could be useful property to have around
         archived: data.archived,
+        userId: data.userId,
         createdOn: parseTimePropertyValue(data.createdAt),
         updatedOn: parseTimePropertyValue(data.updatedAt),
       },
     },
   });
 }
+
+export function createOwnerRoleRelationship(
+  owner: Entity,
+  role: Entity,
+): Relationship {
+  const parentKey = owner._key;
+  const childKey = role._key;
+
+  return createDirectRelationship({
+    _class: RelationshipClass.ASSIGNED,
+    fromKey: parentKey,
+    fromType: Entities.OWNER._type,
+    toKey: childKey,
+    toType: Entities.TEAM._type,
+    properties: {
+      _type: Relationships.OWNER_ASSIGNED_ROLE._type,
+    },
+  });
+}
```

Comment:
I'm curious, why did you decide to create a relationship this way rather than how you've done it in the past?
E.g. like here: https://github.com/creativice/graph-wpengine/blob/master/src/steps/domains/index.ts#L25

Reply:
I saw this https://github.com/creativice/graph-hubspot/blob/bd79dd47698b1b2210ec75f9b744fd38f6f45588/src/steps/teams/converters.ts#L26-L43 and I thought I should just follow the pattern. I have no problem with doing it similar to the WP Engine integration moving forward though if that's what you prefer. =D

Reply:
Yes please do it as you did it before so that it's in the index.ts file + the code is a lot slimmer :slightly_smiling_face:

Reply:
I agree